File: 75. Sort Colors/SortColors.java
class Solution {
    public void sortColors(int[] nums) {
        
        int[] arr = new int[3];

        for (int i=0; i<nums.length; i++) {
            arr[nums[i]] += 1;
        }

        int idx = 0;
        for (int i=0; i<3; i++){
            int val = arr[i];

            while (val > 0) {
                nums[idx++] = i;
                val -= 1;
            }
        }
    }
}

File: 75. Sort Colors/SortColors.py
class Solution:
    def sortColors(self, nums: List[int]) -> None:
        """
        Do not return anything, modify nums in-place instead.
        """
        arr = [0] * 3

        for i in range(len(nums)):
            arr[nums[i]] += 1

        idx = 0

        for i in range(3):
            val = arr[i]

            while (val > 0):
                nums[idx] = i
                val -= 1
                idx += 1

        
